object-cache.php

```diff
@@ -978,9 +1027,11 @@ protected function _should_use_redis_hashes( $group ) {
 	protected function _connect_redis() {
 		global $redis_server;
 
-		if ( ! class_exists( 'Redis' ) ) {
+		// TODO update $redis_server to include 'adapter' key
+		$dependencies_ok = wp_redis_client_check_dependencies();
```

Comment:
I can get behind this! IIRC, `object-cache.php` is loaded before plugins are. What would be the way users could hook into these filters? `add_filter` isn't available to `wp-config.php` without hacking core and mu-plugins wouldn't have been loaded yet?